tt_metal/hw/inc/ethernet/tunneling.h

```diff
@@ -333,6 +340,13 @@ FORCE_INLINE void eth_tunnel_dst_forward_one_cmd(db_cb_config_t *eth_db_cb_confi
 
     routing_info->dst_acked_valid_cmd = 0;
 
+    // Increment available issue slot on remote DST
+    if constexpr(sender_is_issue_path) {
+        *remote_issue_cmd_slots -= 1;
+    } else {
+        *remote_issue_cmd_slots += 1;
```

Comment:
No need for atomic incr?

Reply:
this is local L1 so no need right?

Reply:
as long as no one else is modifying this

Reply:
Yea, no one else is modifying